### PR TITLE
Update v-list-user-packages

### PR DESCRIPTION
**Pull request by codebyoul**
Add "resource limitation" to the list of packages.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
 		"lint-staged": "lint-staged",
 		"format": "prettier --cache --write .",
 		"preinstall": "npx only-allow npm",
-		"prepare": "husky"
+		"prepare": "husky",
+		"hestia": "cd src && ./hst_autocompile.sh --hestia --install '~localsrc'"
```
Comment (AlecRust) [inline package.json]: This comment still applies: https://github.com/hestiacp/hestiacp/pull/4305#discussion_r1501895145

Can you explain your use case here? Are you running [inside a VM](https://hestiacp.com/docs/contributing/development.html#creating-a-virtual-machine-for-development)?

As mentioned the docs will need to be updated with this if it's added. Also `hestia` isn't a very descriptive script name.

Comment (codebyoul) [reply package.json]: > This comment still applies: [#4305 (comment)](https://github.com/hestiacp/hestiacp/pull/4305#discussion_r1501895145)
> 
> Can you explain your use case here? Are you running [inside a VM](https://hestiacp.com/docs/contributing/development.html#creating-a-virtual-machine-for-development)?
> 
> As mentioned the docs will need to be updated with this if it's added. Also `hestia` isn't a very descriptive script name.

There is no need to update the documentation; this command is intended for someone who needs to run a build while developing inside a virtual machine. It's a short command.

Comment (AlecRust) [reply package.json]: > this command is intended for someone who needs to run a build while developing inside a virtual machine

That's specifically [this documentation](https://hestiacp.com/docs/contributing/development.html). Why does it not need to be updated?

When I develop inside the virtual machine as documented (with source code mapped outside the VM) I just press the "up" arrow and press "enter" to run a build. Why is running `npm run hestia` better?

**Review by jaapmarcus — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (codebyoul) [reply package.json]: I don't use 'multipass,' and I can't find the 'up' arrow mentioned. The only instruction provided in the documentation is to run the Hestia build script using the following command: `'./hst_autocompile.sh --hestia --install '~localsrc'.`

Comment (jaapmarcus) [reply package.json]: Up arrow is used to repeat the last "command"  executed...

Comment (codebyoul) [reply package.json]: Ah, okay, I see. You mean the history of commands. I think the convenient way is to add a command in package.json (`npm run command`) as I did, or in composer.json (`composer command`).

**Pull request merged**